helm/opsy/values.yaml

```diff
@@ -0,0 +1,93 @@
+# Default values for opsy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: objectrocket/opsy
+  tag: latest
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - /opsy
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+
+opsy:
+  admin_password:  # This is stored as a secret
```

Comment:
this variable, and all the others.... Helm charts tend to follow the go rules for variable naming. While certainly not required, camel case with lower case first letter is the generally used format: adminPassword, databaseURI, uriPrefix, loggedInPermissions  etc.... it doesn't change the functionality and is certainly not a mandate, just general style in the community

Reply:
In the case of the opsy dict I'm using the same config keys as the opsy config file, which uses snake_case. I think that's the only place I'm using snake_case in this chart.